lib/Data/Time/Clock/Internal/SystemTime.hs

```diff
@@ -1,4 +1,4 @@
-#ifdef mingw32_HOST_OS
```

Comment:
You'll need this beforehand:
```
#include "HsTimeConfig.h"
```

Reply:
Right. I wondered how this failed. Changed it now.